shared/ios/Keybase/Fs.m

```diff
@@ -128,12 +128,12 @@ - (NSDictionary*) setupFs:(BOOL)skipLogFile setupSharedHome:(BOOL)setupSharedHom
   // Setup app level directories
   NSString* levelDBPath = [@"~/Library/Application Support/Keybase/keybase.leveldb" stringByExpandingTildeInPath];
   NSString* chatLevelDBPath = [@"~/Library/Application Support/Keybase/keybase.chat.leveldb" stringByExpandingTildeInPath];
-  NSString* logPath = [@"~/Library/Caches/Keybase" stringByExpandingTildeInPath];
+  NSString* logPath = [@"~/Library/Caches/Keybase/logs" stringByExpandingTildeInPath];
```

Comment:
we need some kind of cleanup of old logs story if we're moving the logs around

Reply:
ah good catch